samples/simple_spellcheck.py

```diff
@@ -0,0 +1,55 @@
+# simple_spellcheck.py
+# Guy Dumais, 2018-08-30
+# Copyright (c) 2018 Element AI. All rights reserved.
+
+import os
+
+from spellchecker.symspell import SymSpell, Verbosity  # import the module
+
+
+def main():
+    # create object
+    initial_capacity = 83000
+    # maximum edit distance per dictionary precalculation
+    max_edit_distance_dictionary = 2
+    prefix_length = 7
+    sym_spell = SymSpell(initial_capacity, max_edit_distance_dictionary,
+                         prefix_length)
+    # load dictionary
+    dictionary_path = os.path.join(os.path.dirname(__file__), '../spellchecker',
+                                   "frequency_dictionary_en_82_765.txt")
+    term_index = 0  # column of the term in the dictionary text file
+    count_index = 1  # column of the term frequency in the dictionary text file
+    if not sym_spell.load_dictionary(dictionary_path, term_index, count_index):
+        print("Dictionary file not found")
+        return
+
+    # lookup suggestions for single-word input strings
+    input_term = "memebers"  # misspelling of "members"
+    # max edit distance per lookup
+    # (max_edit_distance_lookup <= max_edit_distance_dictionary)
+    max_edit_distance_lookup = 2
+    suggestion_verbosity = Verbosity.CLOSEST  # TOP, CLOSEST, ALL
+    suggestions = sym_spell.lookup(input_term, suggestion_verbosity,
+                                   max_edit_distance_lookup)
+    # display suggestion term, term frequency, and edit distance
+    for suggestion in suggestions:
+        print("{}, {}, {}".format(suggestion.term, suggestion.count,
```

Comment:
I think you can just print the suggestion, SuggestItem defines a __str__ that does pretty much the same thing. 

I did change my local copy of the __str__ to this to be more clearer, if you like it you could use it:

```
    def __str__(self):
        return f"{self._term} - Count: {self._count} - Distance: {self._distance}"
```